monitor/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+services:
+  iotex:
+    container_name: iotex
+    image: iotex/iotex-core:v0.10.0
+    restart: on-failure
+    ports:
+      - 8080:8080
+      - 4689:4689
+    volumes:
+      - $IOTEX_HOME/data:/var/data:rw
```

Comment:
if you read the script, the `$IOTEX_HOME` is not read from env, but have a default value and allow user to overwrite it when running the script. So if user doesn't set `$IOTEX_HOME` in env, this path will be wrong. And user will get an error like:
```
ERROR: for iotex  Cannot start service iotex: b'Mounts denied: \r\nThe paths /data and /log and /etc/genesis.yaml and /etc/config.yaml\r\nare not shared from OS X and are not known to Docker.\r\nYou can configure shared paths from Docker -> Preferences... -> File Sharing.\r\nSee https://docs.docker.com/docker-for-mac/osxfs/#namespaces for more info.\r\n.'

ERROR: for iotex  Cannot start service iotex: b'Mounts denied: \r\nThe paths /data and /log and /etc/genesis.yaml and /etc/config.yaml\r\nare not shared from OS X and are not known to Docker.\r\nYou can configure shared paths from Docker -> Preferences... -> File Sharing.\r\nSee https://docs.docker.com/docker-for-mac/osxfs/#namespaces for more info.\r\n.'
```
because `$IOTEX_HOME` is empty in env